src/header/header-buttons.scss

```diff
@@ -0,0 +1,346 @@
+  max-width: 25vw;
+  transition: background-color $iui-speed-fast ease-out;
+  height: 100%;
+  display: flex;
+  overflow: hidden;
+
+  &:hover {
+    @include themed {
+      background-color: rgba(t(iui-color-foreground-body-rgb), t(iui-opacity-6));
+    }
+  }
+
+  &-button:not(:only-child) {
+    padding-right: $iui-xs;
+  }
+
+  &-button-dropdown {
+    padding: 0 $iui-xs;
+  }
+
+  &.iui-active {
+    &,
+    &:hover {
+      // Blue background and darker blue bottom line
+      @include themed {
+        box-shadow: 0 $iui-xxs 0 0 t(iui-color-foreground-primary);
+        background-color: rgba(t(iui-color-foreground-primary-rgb), t(iui-opacity-6));
+      }
+    }
+
+    // Disabled state
+    &.iui-disabled,
+    &[aria-disabled='true'] {
+      &,
+      &:hover {
+        @include themed {
+          background-color: rgba(t(iui-color-foreground-body-rgb), 0.05);
+          box-shadow: 0 $iui-xxs 0 0 rgba(t(iui-color-foreground-body-rgb), t(iui-opacity-4));
+        }
+      }
+    }
+  }
+
+  &.iui-disabled,
+  &[aria-disabled='true'] {
+    cursor: not-allowed;
+
+    &:hover:not(.iui-active) {
+      background-color: transparent;
+    }
+  }
+}
+
+@mixin iui-header-breadcrumb-button {
+  @include iui-reset;
+  @include iui-focus;
+  display: inline-flex;
+  align-items: center;
+  vertical-align: middle;
+  justify-content: center;
+  height: 100%;
+  border-radius: 0;
+  max-width: 25vw;
+  overflow: hidden;
+  padding-left: $iui-sm;
+  padding-right: $iui-s;
+  box-sizing: border-box;
+  -webkit-tap-highlight-color: transparent;
+  transition: background-color $iui-speed-fast ease-out;
+  cursor: pointer;
+  @include themed {
+    color: rgba(t(iui-color-foreground-body-rgb), t(iui-opacity-2));
+  }
+
+  // IE solution
+  > * + * {
+    margin-left: $iui-sm;
+  }
+  @supports (gap: $iui-sm) {
+    gap: $iui-sm;
+
+    > * + * {
+      margin-left: 0;
+    }
+  }
+  // End IE solution
+
+  &,
+  &:hover {
+    background-color: transparent;
+  }
+
+  &-dropdown {
+    padding: 0 $iui-xs;
+
+    &:hover:enabled {
+      @include themed {
+        background-color: rgba(t(iui-color-foreground-body-rgb), t(iui-opacity-6));
+      }
+    }
+  }
+
+  .iui-header-breadcrumb-button-icon {
+    display: inline-flex;
+    padding: $iui-xs;
+    box-sizing: content-box;
+    @include iui-header-breadcrumb-button-icon($iui-icons-default, $iui-border-radius);
+    @include themed {
+      background-color: t(iui-color-background-3);
+      fill: t(iui-icons-color);
+    }
+
+    @media (max-width: 768px) {
+      display: none;
+    }
+  }
+
+  img.iui-header-breadcrumb-button-icon {
+    padding: 0;
+    width: $iui-icons-large;
+    height: $iui-icons-large;
+    object-fit: cover;
+  }
+
+  &.iui-disabled,
+  &:disabled,
+  &[aria-disabled='true'] {
+    cursor: not-allowed;
+
+    svg {
+      @include themed {
+        fill: rgba(t(iui-color-foreground-body-rgb), t(iui-opacity-5));
+      }
+    }
+  }
+
+  @at-root {
```

Comment:
The rules inside this `@at-root` don't belong in this mixin. If you look at the compiled output, you will find that none of these have the `iui-header-breadcrumb-button` class (which is the name of the mixin these are rules in currently)